### PR TITLE
Fix issues in twilio API docs

### PR DESCRIPTION
**Pull request by anupama-pathirage**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

Related Pull Requests (remove if not relevant)
- Pull request 1
- Pull request 2

One line release note: 
- One line describing the feature/improvement/fix made by this PR 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Ballerina Version:
* Operating System:
* Java SDK: 

# Checklist:

### Security checks
 - [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? 
 - [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: twilio/Module.md [hunk 1]
```diff
@@ -27,38 +23,44 @@ If you use a trail account, you may need to verify your recipient phone numbers
...
     auth: {
-        accountSId: accountSId,
-        authToken: authToken
+        accountSId,
+        authToken
     }
 };
 
-twilio:Client twilioClient = new (twilioConfig);
+twilio:Client twilio = new (twilioConfig);
```
Comment (NipunaRanasinghe) [inline twilio/Module.md:51]: Is this (not using the `Client` suffix in client endpoint variable naming) a convention we should be using in all the code samples?

**Review by NipunaRanasinghe — DISMISSED** [review]

File: twilio/Module.md (hunk 1 above)
Comment (anupama-pathirage) [reply twilio/Module.md:51]: Yes, we are using it all new samples we are writing. For ex: refer https://github.com/ballerina-guides/integration-samples/blob/main/github-pull-requests-to-gsheets/main.bal 

So make sure to follow those in upcoming sample writing efforts as well.

**Comment by anupama-pathirage** [comment]
@NipunaRanasinghe Resolved the conflicts and appreciate if you can re-approve.

File: twilio/Module.md (hunk 1 above)
Comment (NipunaRanasinghe) [reply twilio/Module.md]: @anupama-pathirage Ack :) will make sure to follow the same convention in future efforts

**Review by NipunaRanasinghe — APPROVED** [review]

**Pull request merged**